noir-projects/noir-contracts/contracts/token_contract/src/main.nr

```diff
@@ -478,7 +478,7 @@ contract Token {
         let token = Token::at(context.this_address());
 
         // We prepare the transfer.
-        let hiding_point_slot = _prepare_transfer_to_private(to, &mut context, storage);
+        let hiding_point_slot = _prepare_transfer_to_private(from, to, &mut context, storage);
```

Comment:
We now pass the `from` param as it allows us to define the recipient of outgoing.